schemas/documents/shared/claim.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  title: 'Claim',
+  name: 'claim',
+  type: 'document',
+  fields: [
+    {
+      title: 'Description',
+      name: 'description',
+      type: 'string',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Third Party Verified',
+      name: 'thirdPartyVerified',
```

Comment:
@blushi I added a `thirdPartyVerified` boolean because looking at the design we might need it to display a special text+icon.
![image](https://user-images.githubusercontent.com/530644/234799970-c5948833-abb0-4344-95f8-55df59dca08f.png)

Reply:
Ok I will remove it.

Reply:
Done here [073d5e8](https://github.com/regen-network/regen-sanity/pull/34/commits/073d5e80a74ea3bac2092c5d664d7506f971c3d2)